lib/macho/load_commands.rb

```diff
@@ -399,6 +480,19 @@ def initialize(view, cmd, cmdsize, name, timestamp, current_version, compatibili
       @current_version = current_version
       @compatibility_version = compatibility_version
     end
+
+    # @param context [MachO::LoadCcommand::SerializationContext] the context
+    # @return [String] the serialized fields of the load command
+    # @api private
+    def serialize(context)
+      format = Utils.specialize_format(FORMAT, context.endianness)
+      lc_str = SIZEOF # a dylib command can only have one dylib
+      cmdsize = Utils.round(SIZEOF + name.to_s.bytesize + 1, context.cmd_round)
+      npad = cmdsize - SIZEOF - name.to_s.bytesize
+      padded_name = name.to_s + "\x00" * npad
```

Comment:
Sorry, couldn't resist 🙈: It's a comment, I'm not *command*ing you just yet. :joy: